test/torchaudio_unittest/torchscript_consistency_impl.py

```diff
@@ -585,7 +607,7 @@ def test_TimeStretch(self):
         n_freq = 400
         hop_length = 512
         fixed_rate = 1.3
-        tensor = torch.rand((10, 2, n_freq, 10, 2))
+        tensor = torch.rand((10, 2, n_freq, 10, 2), dtype=torch.double)
```

Comment:
Is this change related or necessary?
My understanding is that `_assert_consistency` method will change the dtype/device to appropriate ones so this change has no effect.

Reply:
oh okay removed it!

Reply:
I still see `dtype=torch.double` ...